app/controllers/api/v2/compliance/arf_reports_controller.rb

```diff
@@ -84,6 +88,12 @@ def handle_download_error(error)
           render_error 'standard_error', :status => :internal_error, :locals => { :exception => error }
         end
 
+        def no_proxy_for_host(asset)
+          msg = _('Failed to upload Arf Report, no openscap proxy set for host %s') % asset.host.name
+          logger.error msg
+          render :json => { :result => msg, :id => '' }, :status => :unprocessable_entity
```

Comment:
why the response contain empty id?

Reply:
I suppose I could just omit the id entirely.